models/stg_quickbooks.yml

```diff
@@ -826,6 +844,8 @@ models:
         description: The value of this field will always be set to zero.
       - name: transaction_date
         description: Date when the journal entry was transacted.
+      - name: updated_at
+        description: Timestamp of the date which the journal entry was last updated.
```

Comment:
Since we're making this update I noticed the `source_relation` is not documented for Journal Entry. Can you add this to the documentation.
![image](https://github.com/user-attachments/assets/78e442ff-822e-47eb-92ee-9d688486ae9b)

Reply:
Added